website/content/v1.4/introduction/support-matrix.md

```diff
@@ -6,8 +6,8 @@ description: "Table of supported Talos Linux versions and respective platforms."
 
 | Talos Version                                                                                                  | 1.4                                | 1.3                                |
 |----------------------------------------------------------------------------------------------------------------|------------------------------------|------------------------------------|
-| Release Date                                                                                                   | 2023-03-15                         | 2022-12-15 (1.3.0)                 |
-| End of Community Support                                                                                       | 1.5.0 release (2023-06-15, TBD)    | 1.3.0 release (2022-12-15)         |
+| Release Date                                                                                                   | 2023-04-18                         | 2022-12-15 (1.3.0)                 |
+| End of Community Support                                                                                       | 1.5.0 release (2023-08-15)         | 1.4.0 release (2023-04-18)         |
 | Enterprise Support                                                                                             | [offered by Sidero Labs Inc.](https://www.siderolabs.com/support/) | [offered by Sidero Labs Inc.](https://www.siderolabs.com/support/) |
 | Kubernetes                                                                                                     | 1.27, 1.26, 1.22                   | 1.26, 1.25, 1.24                   |
```

Comment:
the k8s version here should be `1.27, 1.26, 1.25`

Reply:
yes, thanks for noticing that!